shelter/dist/deb/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: shelter
+Section: devel
+Priority: extra
+Maintainer: zhiming hu <zhiming.hu@intel.com>
+Build-Depends: debhelper (>=9)
```

Comment:
You should find a clean ubuntu container and build shelter again and add the Build-Depends please.

Reply:
Add dependence lib into Build-Depends